### PR TITLE
Add detail to nest subscriber instructions

### PR DESCRIPTION
**Pull request by allenporter**
## Proposed change

Add more detail to the nest subscriber instructions and troubleshooting, since it can be tricky to setup correctly or diagnose issues with camera streaming problems (e.g. ensuring the subscriber is set up for the right topic, understanding if messages are being received, or debugging underlying auth problems in the subscriber library, or errors refreshing streaming urls)

## Type of change

- [X] Spelling, grammar or other readability improvements (`current` branch).
- [X] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/nest.markdown [hunk 1]
```diff
@@ -18,6 +18,166 @@ ha_domain: nest
...
+<div class='note warning'>
+New users are not currently able to set up a Works With Nest Developer account. The documentation is preserved here for existing users of the API.
+</div>
+
+
+<details>
+<summary>Click here for documentation for the Legacy Works with Nest API</summary>
+
+The Nest integration is the main integration to integrate all [Nest](https://nest.com/) related platforms. To connect Nest, you will have to [sign up for a developer account](https://developers.nest.com/products) and get a `client_id` and `client_secret`.
+>>>>>>> Update nest troubleshooting instructions
```
Comment (frenck) [inline source/_integrations/nest.markdown]: ?

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by frenck — DISMISSED** [review]
Thanks! 👍

**Comment by allenporter** [comment]
Please take another look.   Apologies!  I didn't notice the duplicate text when viewing locally since it was hidden under the hidden detail, and didn't look at the merge.  Thanks for understanding.

**Review by frenck — APPROVED** [review]
👍

**Pull request merged**